.github/workflows/ruby_tests.yml

```
@@ -0,0 +1,46 @@
+name: Ruby Tests
+on: pull_request
```

If we want then we can run this workflow for both pull_request raised and merged.
```suggestion
on: [push, pull_request]
```